src/Languages/Properties/VariableProperties.ts

```diff
@@ -46,4 +46,29 @@ export class VariableProperties {
      * How to start a variable's type name.
      */
     public typeLeft: string;
+
+    /**
+     * Precedes variable when type is checked negetivly for null.
+     */
+    public isNotNullLeft: string;
+
+    /**
+     * Precedes variable when type is checked positivly for null.
+     */
+    public isNullLeft: string;
+
+    /**
+     * Equality operator.
+     */
+    public isNullMiddle: string;
+
+    /**
+     * Inequality operator.
```

Comment:
The descriptions for `isNullMiddle` and `isNotNullMiddle` are pretty vague. Please be a bit more informative.